.github/workflows/train_operator_kind_test.yaml

```diff
@@ -29,3 +29,8 @@ jobs:
         kubectl create ns kubeflow
         kustomize build overlays/kubeflow | kubectl apply -f -
         kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout 180s
+
+    - name: Create a TFJob
+      run: |
+        kubectl create -f tests/gh-actions/kf-objects/tfjob.yaml
+        kubectl wait --for=condition=Running tfjobs tfjob-simple -n kubeflow --timeout 300s
```

Comment:
Doesn't `kubectl wait --for=condition=Running` mean that the script will wait until the TFJob starts running?

We should instead wait until it completes

Reply:
I changed the condition check to `Succeeded`.